mittab/apps/tab/models.py

```diff
@@ -270,6 +270,12 @@ def debaters_display(self):
             return ", ".join([debater.name for debater in self.debaters.all()])
         return ""
 
+    def get_team_code(self):
```

Comment:
OOC, do you know this situations where this isn't being set? Would prefer to just do that proactively where we could (potentially) generate and save all team-codes at once, rather than doing it ad-hoc which could result in 100+ distinct SQL writes if this is called from a list view

Also: slightly prefer naming something like `get_or_create_team_code` to indicate the behavior

Reply:
I slightly suspect the test data fixtures may be the only case where we want team codes but dont have them